test/RPCSession.cpp

```diff
@@ -187,7 +188,7 @@ string RPCSession::eth_getStorageRoot(string const& _address, string const& _blo
 
 void RPCSession::personal_unlockAccount(string const& _address, string const& _password, int _duration)
 {
-	rpcCall("personal_unlockAccount", { quote(_address), quote(_password), to_string(_duration) });
+	BOOST_CHECK(rpcCall("personal_unlockAccount", { quote(_address), quote(_password), to_string(_duration) }) == true);
```

Comment:
Is the `== true` required?
Furthermore, if this call does not work, it does not make much sense in continuing the test, does it? So `BOOST_REQUIRE` might be better here.